packages/fxa-content-server/app/scripts/views/mixins/password-strength-mixin.js

```diff
@@ -32,6 +32,11 @@ export default function(config = {}) {
           // problems in the reset-password screen when using the recovery key
           return;
         }
+        if (window.matchMedia('(max-width: 960px)').matches) {
```

Comment:
Eh? I think it will make @ryanfeeley sad to disable the balloon universally on mobile devices. The balloon is supposed to go away once the password is valid, is that broken? See also https://github.com/mozilla/fxa/issues/1470.

Reply:
It's broken on TV because it prevents you from completing the form at all. I'm happy to fix it but my priority is to unblock them and follow up with a working balloon

Reply:
Can we UA sniff? Gross, but doesn't kill the feature for everyone.

Reply:
Seems broken for everyone to me with #1470

Reply:
This UI needs some TLC. The repeat password message should disappear when the basic requirements are met (for the second field that means the password is repeated). Original design: https://mozilla.invisionapp.com/share/5UM3J3TXWMR#/316338169_Artboard

Reply:
> Seems broken for everyone to me with #1470

In the bugzilla bug I asked the user if the password bubble goes away if the requirements are met. Yes, yes they do.

> I just tried it again, and it does indeed go away when the requirements are met. I still maintain the interface is surprising. Covering form elements with a little box like that so you cannot click on them is not a common way to communicate that elements are not available or enabled. Perhaps showing those elements but graying them out would be a more effective way to communicate the message. However, if you don't consider this a bug, I suppose it's not a bug. Thanks for taking a look.

Reply:
There isn't anything very unique about the ua string on TV.
`Mozilla/5.0 (Linux; Android 7.1.2) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Focus/4.2-LAT1 Chrome/59.0.3071.125 Mobile Safari/537.36` (`Focus` is the closest thing worth filtering on I suppose)

I don't see anything wrong with disabling it on mobile until it's fixed. It's obviously causing enough confusion for someone to file a bug, even if it technically works.

Reply:
If we merge this after the train I may even have the bubble fixed before the next train. Really, I just want this on `latest`

Reply:
> If we merge this after the train I may even have the bubble fixed before the next train. Really, I just want this on latest

You've convinced me! r+